lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_state_machine.rb

```diff
@@ -8,14 +8,16 @@ def state_runnable?(f)
       true
     end
 
-    def initialize_state_maxima_metadata
+    def initialize_state_maxima_metadata(f)
```

Comment:
I realize you are just passing it along, but `f` is a horrible parameter name.  (I am assuming it stands for `field`.) 
 Can we start using a more descriptive name here so we have some idea what the parameter contains.